vllm/yichat.py

```diff
@@ -0,0 +1,48 @@
+from openai import OpenAI
```

Comment:
make sure to add either a requirements.txt and instructions for it, I didn't have openai in my virtualenv